src/components/ModalWindow/index.js

```diff
@@ -20,18 +22,10 @@ const ModalWindow = ({ toggleModal, Component, member }) => {
 ModalWindow.propTypes = {
   toggleModal: PropTypes.func.isRequired,
   Component: PropTypes.elementType.isRequired,
-  member: PropTypes.exact({
-    tasks: PropTypes.array,
-    id: PropTypes.string.isRequired,
-    age: PropTypes.number.isRequired,
-    direction: PropTypes.string.isRequired,
-    education: PropTypes.string.isRequired,
-    email: PropTypes.string.isRequired,
-    firstName: PropTypes.string.isRequired,
-    lastName: PropTypes.string.isRequired,
-    startDate: PropTypes.number.isRequired,
-    progress: PropTypes.array,
-  }),
+  member: PropTypes.object,
+  typeForm: PropTypes.string.isRequired,
+  task: PropTypes.object,
+  track: PropTypes.object,
```

Comment:
is everything ok here ?

Reply:
after using props.children it became like this:
```suggestion
  ModalWindow.propTypes = {
  toggleModal: PropTypes.func.isRequired,
};
```